ert3_examples/spe1/run_demo

```diff
@@ -7,11 +7,7 @@
 
 set -e
 
-# TODO: This is a hack because of the lack of blob records. Hence, we currently
-# copy the datafile to the compute nodes and this environment variable makes
-# the workspace transportable.
-export SPE1_WORKSPACE_ROOT=$(pwd)
-
+ert3 record load example_data resources/SPE1CASE2.DATA.jinja2 --blob-record
```

Comment:
While here, when loading it into storage the case is highly relevant, as then one could potentially load multiple cases into storage. Hence, I suggest here that we replace `example_data` with `spe1case2_template`.